constants/index.js

```diff
@@ -1,3 +1,161 @@
+const footerColumns = [
+  {
```

Comment:
Same, move it to a constant. Furthermore, I'm seeing some repeated data from another place. Probably you just need the src for telegram, link, icon in one place and use it multiple times like here.

Reply:
Nevermind, just realised this was the constant file lol :sweat_smile: